AndroidAssignment/app/src/main/java/org/sopt/data/local/SOPTSharedPreference.kt

```diff
@@ -0,0 +1,40 @@
+package org.sopt.data.local
+
+import android.content.Context
+import android.content.SharedPreferences
+
+object SOPTSharedPreference {
+    private const val AUTO_LOGIN = "AUTO_LOGIN"
+    private const val ID_PASSWORD_EXIST = "ID_PASSWORD_EXIST"
+    private const val USER_NAME = "USER_NAME"
+
+    lateinit var preferences: SharedPreferences
+
+    fun init(context: Context) {
+        preferences = context.getSharedPreferences(context.packageName, Context.MODE_PRIVATE)
+    }
+
+    fun getAutoLogin(): Boolean {
+        return preferences.getBoolean(AUTO_LOGIN, false)
+    }
+
+    fun setAutoLogin(value: Boolean) {
+        preferences.edit().putBoolean(AUTO_LOGIN, value).apply()
+    }
+
+    fun getIdPasswordExist(): Boolean {
+        return preferences.getBoolean(ID_PASSWORD_EXIST, false)
+    }
+
+    fun setIdPasswordExist(value: Boolean) {
+        preferences.edit().putBoolean(ID_PASSWORD_EXIST, value).apply()
```

Comment:
안드로이드 KTX를 쓰면 조금 더 깔끔하게 작성 가능함
```suggestion
        preferences.edit { putBoolean(ID_PASSWORD_EXIST, value) }
```

Reply:
이야.. 안드파트 끝판왕이 오셨군요.. 코드 리뷰 받게되서 영광입니다.. 감동받아서 손 발이 부들거리네요 ㅋㅋㅋㅋㅋㅋㅋ(화난거 아님) 암튼 **Android Extension** 어려워서 잘 손이 안가게 되던데 확실히 앞으로는 도전해봐야 될 것 같아요.. 현우님이 수정해주신 코드가 정말 훨씬 깔끔해보이네요..! 수정해보겠습니다